index.html

```diff
@@ -222,5 +222,37 @@ <h2>Skills</h2>
             </div>
         </div>
     </section>
+    <section class="contact-form" id="contact">
+        <div class="contact-text">
+            <p>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+        </div>
+        <div class="contact-area">
+            <form action="https://formspree.io/f/xvolllzo" method="post" id="frmContact">
+              <label for="name">
+                <input type="text" name="name" id="name" maxlength="30" class="contact-input" placeholder="Enter your names..."
+                required/>
+              </label>
+              <label for="email">
+                <input type="email" name="email" id="email" class="contact-input" placeholder="Enter your gmail..."
+                required/>
+              </label>
+              <label for="text">
+                <textarea name="text" id="text" maxlength="500" class="contact-input" placeholder="Enter text here" cols="23" rows="5"></textarea>
+              </label>
+                <input id="btn" type="submit" class="submit-btn dark-font" value="Get in Touch" />
+            </form>
+          </div>
+    </section>
+    <section class="footer">
+        <ul class="f-unordered">
+            <li class="f-list"><a href="" aria-label="github"><img src="./img/Vector-github.png" alt="github" class="img-icon"/></a></li>
+            <li class="f-list"><a href="" aria-label="linkedin"><img src="./img/vector-linkedin.png" alt="linkedin" class="img-icon"/></a></li>
+            <li class="f-list"><a href="" aria-label="medium"><img src="./img/Vector.png" alt="medium" class="img-icon"/></a></li>
+            <li class="f-list"><a href="" aria-label="angelist"><img src="./img/Vector-angelist.png" alt="angelist" class="img-icon"/></a></li>
+            <li class="f-list"><a href="" aria-label="twitter"><img src="./img/Vector-twitter.png" alt="twitter" class="img-icon"/></a></li>
+        </ul>
```

Comment:
- [ ]  Kindly maintain the project's Figma design in terms of colors, fonts, etc. Your footer social media icons currently do not meet all the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md). Kindly fix this.

**This is how your project currently looks like:**
![image](https://user-images.githubusercontent.com/86318284/177408896-0d94783a-9b82-4eb9-88d3-1630966562bc.png)


**This is the desired Figma design:**
![image](https://user-images.githubusercontent.com/86318284/177408969-8b1dca88-7e4a-4419-88a5-4083010cfa42.png)